docs/src/man/basics.md

```diff
@@ -0,0 +1,1388 @@
+  994 │ male        3  own
+  995 │ male        2  own
+  996 │ female      1  own
+  997 │ male        3  own
+  998 │ male        2  own
+  999 │ male        2  free
+ 1000 │ male        2  own
+               985 rows omitted
+
+julia> german[:, Cols("Age", Between("Sex", "Job"))] # presents `Age` column as well as `Sex` and `Job`
+1000×3 DataFrame
+  Row │ Age    Sex     Job
+      │ Int64  String  Int64
+──────┼──────────────────────
+    1 │     1  male        2
+    2 │     1  female      2
+    3 │    49  male        1
+    4 │    45  male        2
+    5 │    53  male        2
+    6 │    35  male        1
+    7 │    53  male        2
+    8 │    35  male        3
+  ⋮   │   ⋮      ⋮       ⋮
+  994 │    30  male        3
+  995 │    50  male        2
+  996 │    31  female      1
+  997 │    40  male        3
+  998 │    38  male        2
+  999 │    23  male        2
+ 1000 │    27  male        2
+             985 rows omitted
+
+julia> german[:, Cols("Age", Not("Sex"))] # Shows all column except `Sex` because of `Not` operation
+1000×9 DataFrame
+  Row │ Age    id     Job    Housing  Saving accounts  Checking account  Credi ⋯
+      │ Int64  Int64  Int64  String   String           String            Int64 ⋯
+──────┼─────────────────────────────────────────────────────────────────────────
+    1 │     1      1      2  own      NA               little                  ⋯
+    2 │     1      1      2  own      little           moderate
+    3 │    49      2      1  own      little           NA
+    4 │    45      3      2  free     little           little
+    5 │    53      4      2  free     little           little                  ⋯
+    6 │    35      5      1  free     NA               NA
+    7 │    53      6      2  own      quite rich       NA
+    8 │    35      7      3  rent     little           moderate
+  ⋮   │   ⋮      ⋮      ⋮       ⋮            ⋮                ⋮                ⋱
+  994 │    30    993      3  own      little           little                  ⋯
+  995 │    50    994      2  own      NA               NA
+  996 │    31    995      1  own      little           NA
+  997 │    40    996      3  own      little           little
+  998 │    38    997      2  own      little           NA                      ⋯
+  999 │    23    998      2  free     little           little
+ 1000 │    27    999      2  own      moderate         moderate
+                                                  3 columns and 985 rows omitted
+```
+
+In the above example `german[:, Cols("Age", Not("Sex"))]`, `Age` column will come first after 
+removing `sex` column. For more understanding if we will follow this example `german[:, Cols("Job", Not("Sex"))]`
+then `Job` column will be placed first and then the remaining order will be same after removing `Sex` column.
+
+You can also use `Regex` to select columns and `Not` from [InvertedIndices.jl](https://github.com/mbauman/InvertedIndices.jl) 
+both to select rows and columns:
```

Comment:
mixing two concepts at the same time unnecessarily.

Reply:
this is a technically correct code, but incorrect in a sense that you show two unrelated things in one example which might confuse the reader.